pkg/backingstore/reconciler.go

```diff
@@ -890,6 +890,20 @@ func (r *Reconciler) reconcileExistingPods(podsList *corev1.PodList) error {
 	noneAttachingAgents := 0
 	failedAttachingAgents := 0
 	for _, pod := range podsList.Items {
+
+		// GAP: reconcile proxy env vars.
+		// The only thing that can be changed on a pod spec is the image
+		// trying to change anything else, including env vars, will create a panic.
+		// Deployment and Statefulset are handling this by taking down the pod and
+		// starting a new one.
+		// In order to support reconciling changes to the proxy env we need to mimic
+		// this behaviour which is not trival in the case of agent pods.
+
+		// var c = &pod.Spec.Containers[0]
```

Comment:
for now, maybe log if there is a mismatch between the pod and ther operator envs

Reply:
done